src/components/About.jsx

```diff
@@ -1,9 +1,10 @@
 import React from "react";
+import { Card } from "react-bootstrap";
+import { FaMailBulk, FaLinkedin, FaGithub } from "react-icons/fa";
+import "./About.css";
+//Assets
 import Inaiapicture from "../assets/images/inaia.jpeg";
 import Qrcode from "../assets/images/qrcode.png";

```

Comment:
What data type is this import? Is it a string? If so, we'd consider it a "constant" and convention would say that this and the other constants here should be `QR_CODE`. See more on this here https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/